geodata/models/imagery/base.py

```diff
@@ -69,6 +85,9 @@ class RasterEntry(ImageSet, SpatialEntry, TaskEventMixin):
 
     """
 
+    def __str__(self):
+        return f'{self.name} ({self.id} - {type(self)}'
+
```

Comment:
Perhaps we should move this to `ImageSet` so it also has this functionality and `RasterEntry` will inherit it

Reply:
Will do.